src/accounts/molecules/ConnectionError.tsx

```diff
@@ -0,0 +1,31 @@
+import connectionImage from "../assets/connecting.png";
+import { TP } from "../../common/atoms/THeadings";
+import TImage from "../../common/atoms/TImage";
+import settings from "../../common/settings";
+
+export default function ConnectionError({ username }: { username?: string }) {
+  return (
+    <div className="margin-top">
+      <TImage src={connectionImage} alt="Connecting Phone-PC" />
+
+      {username ? (
+        <>
+          <TP className="margin-top">
+            Could not connect to the mobile app of the user <u>{username}</u>.
+          </TP>
+          <TP>
+            Please open {settings.config.appName} on your phone and login as{" "}
+            <u>{username}</u>.
+          </TP>
+        </>
+      ) : (
+        <>
+          <TP className="margin-top">
+            Could not connect to the {settings.config.appName}.
+          </TP>
+          <TP>Please try again.</TP>
```

Comment:
In connection error shouldn't we redirect user back to login?

Reply:
how about we give a button/link that lets them go back?
Automatically might be confusing.

Reply:
yess that is the same thing i was thinking about or we can make "Please try again" clickable

Reply:
done